doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -1341,14 +1350,19 @@ A context registration is represented by a JSON object with the following fields
 corresponds to the URL of the provider service.
 * `duration`: Duration of the registration in ISO8601 format. Default duration is infinite.
 
-A `subject` contains the following subfields:
+The `subject` support two variants, to register single entity id/pattern or to register a list of them. A subject for
+registering a single id/pattern has the following subfields (`id`/`idPattern` or `type` must be present):
 
-* `id` or `idPattern` : Id or pattern of the affected entities (optional). Both cannot be used at the same time.
-* `entity` : Id or pattern id which identifies the affected entities (optional).
-* `type` : Type of the affected entities (optional).
+* `id` or `idPattern`: Id or pattern of the affected entities (optional). Both cannot be used at the same time.
+* `type`: Type of the affected entities (optional).
 * `attributes`: List of attributes to be provided (if not specified it would mean all).
 
-`id`/`idPattern` or `type` must be present.
+A `subject` for registering a list of ids/patterns has the following subfields (`entities` must be present and not empty):
+
+* `entities`: A list of objects, each one composed of the following subfields (`id`/`idPattern` or type must be present):
+ * `id` or `idPattern`: Id or pattern of the affected entities (optional). Both cannot be used at the same time.
+ * `type`: Type of the affected entities (optional).
+* `attributes`: List of attributes to be provided (if not specified it would mean all).
 
```

Comment:
They are indirectly needed as the closure in the scope definition of the RPC POJ operations related with subscriptions/registrations needs that the RESTful oepration for subscription/registration allow also the posibility of registering/subscribing to list of entities.